File: cop3223H/frame.c
//Federico Baron
//8/24/18
//Calculates the area of a picture frame.

#include <stdio.h>
int main(){

	//prompts user for frame dimensions and saves them as integers
	int blen, bwidth, slen, swidth;
	printf("Enter the dimensions of the outside of the frame.\n");
	scanf("%d%d", &blen, &bwidth);
	printf("Enter the dimensions of the inside of the frame\n");
	scanf("%d%d", &slen, &swidth);

	//defines inner and outer frames
	int outarea = blen * bwidth;
	int inarea = slen * swidth;

	//The difference is the frame area
	int framearea = outarea - inarea;

	//Present
	printf("The area of the frame is %d.\n", framearea);

	return 0;
}

File: cop3223H/frame.py
#Federico Baron
#8/24/18
#Calculates area of frame

#Get the dimensions of the frame (inner and outer)
blen = int(input("What is the outside length of the frame?"))
bwidth = int(input("What is the outside width of the frame?"))
slen = int(input("What is the inner length of the frame?"))
swidth = int(input("What is the inside width of the frame?"))

#Calculate and output the result
print("The area of the frame is", blen*bwidth - slen*swidth)

# / is real number or floating point division
# // is int divisions
